src/app/core/components/navigation/navigation.component.scss

```diff
@@ -0,0 +1,14 @@
+@import 'mixines';
```

Comment:
seems you do not use any mixin or variable here

Reply:
I've removed them